apps/api/src/app/organization/usecases/create-organization/create-organization.usecase.ts

```diff
@@ -60,7 +68,18 @@ export class CreateOrganization {
       })
     );
 
-    await this.createEnvironmentUsecase.execute(
+    await this.createNovuIntegrations.execute(
+      CreateNovuIntegrationsCommand.create({
+        environmentId: devEnv._id,
+        organizationId: devEnv._organizationId,
+        userId: user._id,
+        name: devEnv.name,
+      })
+    );
+
+    await this.createDefaultLayout(devEnv, user._id);
+
```

Comment:
Please revert all of those changes, the problem is with the new layout json structure